### ISSUE TITLE
Unexpected token export

### ISSUE THREAD

**Issue by vmanchev** [souporserious/react-popper#96]
As part of reactstrap, we are facing this error, when running jest tests. Any help will be appreciated.

```
 /Users/venelin/projects/stylbum-web-v2/node_modules/react-popper/lib/react-popper.js:1
    ({"Object.<anonymous>":function(module,exports,require,__dirname,__filename,global,jest){export { default as Manager } from './Manager';
                                                                                             ^^^^^^

    SyntaxError: Unexpected token export

      at ScriptTransformer._transformAndBuildScript (node_modules/jest-runtime/build/ScriptTransformer.js:289:17)
      at Object.<anonymous> (node_modules/reactstrap/dist/reactstrap.cjs.js:14:19)
      at Object.<anonymous> (src/Auth/Login/Login.jsx:5:19)
```

https://github.com/reactstrap/reactstrap/issues/872

**Comment by revolunet** [comment]
same here

**Comment by nickmccurdy** [comment]
I just got this when installing `reactstrap@5.0.0-beta.2`. I'd appreciate it if anyone knows a workaround, this is blocking me from porting a site to Bootstrap 4.

**Comment by sinxwal** [comment]
@nickmccurdy Use reactstrap": "5.0.0-beta" version for a short.

**Comment by nickmccurdy** [comment]
@sinxwal I can confirm `5.0.0-beta` works for me, thanks!

**Comment by Justkant** [comment]
Hi, maybe related to #93

**Comment by halntit** [comment]
Thank you @nickmccurdy, that version really works and save me a lot of time

**Comment by FezVrasta** [comment]
This should be fixed in 0.8.3
